docs/index.rst

```diff
@@ -80,65 +80,72 @@ The best way to get started with Parcels is to have a look at the Jupyter notebo
 
-    source $HOME/miniconda2/bin/activate root
-    conda env create -f environment.yml -n py2_parcels
-    source $HOME/miniconda2/bin/activate py2_parcels
-    pip install git+https://github.com/OceanParcels/parcels.git@master
+2. Start a terminal (Linux / macOS) or the Anaconda prompt (Windows). Activate the root (or base) environment of your Miniconda and create an environment containing Parcels, all its essential dependencies, and the nice-to-have Jupyter and Basemap package::
 
-3. Validate the installation by downloading the `example_peninsula.py <https://raw.githubusercontent.com/OceanParcels/parcels/master/parcels/examples/example_peninsula.py>`_ file, navigate to the directory where you saved that file, and then run::
+    source $HOME/miniconda2/bin/activate root  # Linux / macOS
+    activate root                              # Windows
 
-    python example_peninsula.py --fieldset 100 100
+    conda create -n py2_parcels -c conda-forge parcels jupyter basemap basemap-data-hires
+
+3. Activate the newly created Parcels environment, get a copy of the the Parcels tutorials and examples, and run the simplest of the examples to validate that you have a working Parcels setup::
 
-4. Optionally, if you want to run all the examples and tutorials, you can get tutorials and example data by running::
+    source $HOME/miniconda2/bin/activate py2_parcels  # Linux / macOS
+    activate py2_parcels                              # Windows
 
     parcels_get_examples parcels_examples
     cd parcels_examples
+
+    python example_peninsula.py --fieldset 100 100
+
+4. Optionally, if you want to run all the examples and tutorials, start Jupyter and open the tutorial notebooks::
+
     jupyter notebook
 
 5. The next time you start a terminal and want to work with Parcels, activate the environment with::
 
-    source $HOME/miniconda2/bin/activate py2_parcels
+    source $HOME/miniconda2/bin/activate py2_parcels  # Linux / macOS
+    activate py2_parcels                              # Windows
+
+
+.. _installing-arbitrary-Git-reference:
+
+Installing a non-released version of Parcels
+============================================
 
-An alternative, more advanced option is to use an existing netCDF installation and
-C compiler, ``git clone`` the `master branch of Parcels
-<https://github.com/OceanParcels/parcels>`_
-and ``pip install`` the dependencies in the `environment.yml <https://raw.githubusercontent.com/OceanParcels/parcels/master/environment.yml>`_ file.
+There might be cases where you want to install a version of Parcels that has not been released yet.  (Perhaps, if you want to use a bleeding-edge feature which already is included on Github, but not in the conda-forge package.)
 
-.. _installing-windows:
+Then, just after step 2 of :ref:`installing-parcels` above, remove the conda-forge package again, and use Pip to install Parcels from Github::
```

Comment:
Do we need to say how to remove the parcels package from conda?

Reply:
Oh. Yes, I forgot to add a `conda remove parcels`.

Note that unlike package managers like apt or brew, conda does not remember where all the dependencies came from and really just removes the parcels package here.